reusable_components_nlp/README.md

```diff
@@ -0,0 +1,989 @@
+
+
+```python
+!helm install seldon-core-operator --namespace kubeflow --repo https://storage.googleapis.com/seldon-charts
+```
+
+    NAME:   old-mastiff
+    LAST DEPLOYED: Tue May 28 11:47:35 2019
+    NAMESPACE: kubeflow
+    STATUS: DEPLOYED
+    
+    RESOURCES:
+    ==> v1/ClusterRole
+    NAME                          AGE
+    seldon-operator-manager-role  0s
+    
+    ==> v1/ClusterRoleBinding
+    NAME                                 AGE
+    seldon-operator-manager-rolebinding  0s
+    
+    ==> v1/Pod(related)
+    NAME                                  READY  STATUS             RESTARTS  AGE
+    seldon-operator-controller-manager-0  0/1    ContainerCreating  0         0s
+    
+    ==> v1/Secret
+    NAME                                   TYPE    DATA  AGE
+    seldon-operator-webhook-server-secret  Opaque  0     0s
+    
+    ==> v1/Service
+    NAME                                        TYPE       CLUSTER-IP     EXTERNAL-IP  PORT(S)  AGE
+    seldon-operator-controller-manager-service  ClusterIP  10.101.228.36  <none>       443/TCP  0s
+    
+    ==> v1/StatefulSet
+    NAME                                READY  AGE
+    seldon-operator-controller-manager  0/1    0s
+    
+    ==> v1beta1/CustomResourceDefinition
+    NAME                                         AGE
+    seldondeployments.machinelearning.seldon.io  0s
+    
+    
+    NOTES:
+    NOTES: TODO
+    
+    
+
+
+Check all the Seldon Deployment is running
+
+
+```python
+!kubectl get pod -n kubeflow | grep seldon
+```
+
+    seldon-operator-controller-manager-0                       1/1     Running   1          6s
+
+
+### Temporary fix for Argo image
+
+At the time of writing we need to make some updates in the Argo images with the following commands below.
+
+(This basically changes the images to the latest ones, otherwise we will get an error when we attach the volume)
+
+
+
+```python
+!kubectl -n kubeflow patch deployments. workflow-controller --patch '{"spec": {"template": {"spec": {"containers": [{"name": "workflow-controller", "image": "argoproj/workflow-controller:v2.3.0-rc3"}]}}}}'
+!kubectl -n kubeflow patch deployments. ml-pipeline --patch '{"spec": {"template": {"spec": {"containers": [{"name": "ml-pipeline-api-server", "image": "elikatsis/ml-pipeline-api-server:0.1.18-pick-1289"}]}}}}'
+# !kubectl -n kubeflow patch configmaps workflow-controller-configmap --patch '{"data": {"config": "{ executorImage: argoproj/argoexec:v2.3.0-rc3,artifactRepository:{s3: {bucket: mlpipeline,keyPrefix: artifacts,endpoint: minio-service.kubeflow:9000,insecure: true,accessKeySecret: {name: mlpipeline-minio-artifact,key: accesskey},secretKeySecret: {name: mlpipeline-minio-artifact,key: secretkey}}}}" }}'
+```
+
+    deployment.extensions/workflow-controller patched
+    deployment.extensions/ml-pipeline patched
+
+
+The last command you need to run actually needs to be manual as the patch cannot change configmap contents directly
```

Comment:
Consider providing a file with this change that the user can apply

Reply:
Oh like providing the whole configmaps file? I had difficulties actually trying to do the change in a different way than an edit (hence why I'm not even using patch as previously). This unfortunately is a quickfix that should will hopefully be avoided after kubectl upgrades to the next version of argo.